backend/server/routers/user.py

```diff
@@ -99,6 +99,10 @@ def default_cs_user() -> Storage:
 def toggle_summer_term(token: str = DUMMY_TOKEN):
```

Comment:
another thing, I suggest using auth bearer token header to get the token from the request - also reflect this in the FE request too -> much more standard practice that params